src/modules/deposit/service.ts

```diff
@@ -132,30 +202,10 @@ export class DepositService {
   public async getDepositsV2(query: GetDepositsV2Query) {
     const limit = parseInt(query.limit ?? "10");
     const offset = parseInt(query.offset ?? "0");
-    let queryBuilder = this.depositRepository.createQueryBuilder("d");
-    queryBuilder = queryBuilder.where("d.depositDate is not null");
-
-    if (query.status) {
-      queryBuilder = queryBuilder.andWhere("d.status = :status", { status: query.status });
-    }
-
-    if (query.originChainId) {
-      queryBuilder = queryBuilder.andWhere("d.sourceChainId = :sourceChainId", { sourceChainId: query.originChainId });
-    }
 
-    if (query.destinationChainId) {
-      queryBuilder = queryBuilder.andWhere("d.destinationChainId = :destinationChainId", {
-        destinationChainId: query.destinationChainId,
-      });
-    }
-
-    if (query.tokenAddress) {
-      queryBuilder = queryBuilder.andWhere("d.tokenAddr = :tokenAddr", { tokenAddr: query.tokenAddress });
-    }
+    let queryBuilder = this.depositRepository.createQueryBuilder("d");
+    queryBuilder = this._getFilteredDepositsQuery(queryBuilder, query);
 
-    queryBuilder = queryBuilder.orderBy("d.depositDate", "DESC");
-    queryBuilder = queryBuilder.take(limit);
-    queryBuilder = queryBuilder.skip(offset);
```

Comment:
These 3 should not be removed, right?

Reply:
https://github.com/across-protocol/scraper-api/pull/279/commits/a44ac9d35236845a1fc0a7d5ac196bb24c250ac3